group.go

```diff
@@ -12,113 +12,113 @@ type Group struct {
 	app    *App
 }
 
-// Group ...
-func (app *App) Group(prefix string, args ...interface{}) *Group {
-	if len(args) > 0 {
-		app.register("USE", prefix, args...)
+// Group : https://fiber.wiki/application#group
+func (app *App) Group(prefix string, handlers ...interface{}) *Group {
+	if len(handlers) > 0 {
+		app.register("USE", prefix, handlers...)
 	}
 	return &Group{
 		prefix: prefix,
 		app:    app,
 	}
 }
 
-// Group ...
-func (grp *Group) Group(newPrfx string, args ...interface{}) *Group {
-	var prefix = grp.prefix
-	if len(newPrfx) > 0 && newPrfx[0] != '/' && newPrfx[0] != '*' {
-		newPrfx = "/" + newPrfx
+// Group : https://fiber.wiki/application#group
+func (grp *Group) Group(prefix string, handlers ...interface{}) *Group {
+	var oldPrefix = grp.prefix
+	if len(prefix) > 0 && prefix[0] != '/' && prefix[0] != '*' {
```

Comment:
Will there be unexplained behaviour if somebody will pass `*/abrakadabra` as `prefix`? Maybe compare entire `prefix` with `*`?

Reply:
Our current wildcard logic is that everything behind * is irrelevant, everything behind a `*` is not matched.

Reply:
Okay 👌